src/_hooks/useValidateSchema.ts

```diff
@@ -17,14 +17,14 @@ export function useValidateSchema<T>(
       return;
     }
     schema
-      .validate(value, { strict: true, abortEarly, recursive }) // to avoid taking a useEffect dependency on validateOpts that is an object
+      .validate(value, { strict: true, abortEarly: abortEarly ?? true }) // to avoid taking a useEffect dependency on validateOpts that is an object
```

Comment:
This should do the same assuming abortEarly is a boolean.
```suggestion
      .validate(value, { strict: true, abortEarly: abortEarly !== false }) // to avoid taking a useEffect dependency on validateOpts that is an object
```

Reply:
It may just be my C# background (where we've had `?.` and `??` for ages), but the nullish coalescing version is clearer for me. I sometimes call it the "fallback operator". I can understand yours as well of course, but it requires some thinking on how the thing works and what the default is.